Plugins/BuildServerIntegration/AppVeyorIntegration/AppVeyorAdapter.cs

```diff
@@ -434,9 +434,14 @@ private void UpdateDescription(AppVeyorBuildInfo buildDetails, CancellationToken
 
         private static long GetBuildDuration(JToken buildData)
         {
-            var startTime = buildData["started"].ToObject<DateTime>();
-            var updateTime = buildData["updated"].ToObject<DateTime>();
-            return (long)(updateTime - startTime).TotalMilliseconds;
+            var startTime = buildData["started"]?.ToObject<DateTime>();
```

Comment:
The log in #8150 has 'created', should be used first

```suggestion
            var startTime = buildData["started"]?.ToObject<DateTime>() ?? buildData["created"]?.ToObject<DateTime>();
```

Reply:
indeed. done.